redcap/project.py

```diff
@@ -149,6 +157,423 @@ def _call_api(self, payload, typpe, **kwargs):
         rcr = RCRequest(self.url, payload, typpe)
         return rcr.execute(**request_kwargs)
 
+    def __toggle_dataframe(self,format):
+        ret_format = format
+        if format == 'df':
+            try:
+                from StringIO import StringIO
+            except ImportError:
+                from io import StringIO
+            from pandas import read_csv
+
+            # make accessible elswhere
+            global StringIO
+            global read_csv
+            ret_format = 'csv'
+        return ret_format
+
+    def export_project(self, format='json',df_kwargs=None):
+        """
+        Export the project's information
+
+        Parameters
+        ----------
+        format : (``'json'``), ``'csv'``, ``'xml'``
+            Return the project information,
+            csv or xml, ``'df''`` will return a ``pandas.DataFrame``
+        df_kwargs : dict
+            Passed to pandas.read_csv to control construction of
+            returned DataFrame
+
+        Returns
+        -------
+        project: list, str, ``pandas.DataFrame``
+            project information
+        """
+
+        # Check for dataframe usage
+        ret_format = self.__toggle_dataframe(format)
+
+        pl = self.__basepl('project')
+        response, _ = self._call_api(pl, 'exp_project')
+
+        if format in ('json', 'csv', 'xml'):
+            return response
+        elif format == 'df':
+            if not df_kwargs:
+                return read_csv(StringIO(response))
+            else:
+                return read_csv(StringIO(response), **df_kwargs)   
+
+    def export_report(self, report_id, format='json',raw_or_label='raw',raw_or_label_headers='raw',export_checkbox_labels=False,df_kwargs=None):
```

Comment:
Please include a space after the `,` in lists, function arguments, dictionaries, etc.